sourcemash/frontend/static/logotext.svg

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
You can resize SVG images easily - that’s the whole point. Not sure what you mean….